webpack/scenes/ContentViews/__tests__/contentViewTable.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { render, fireEvent, within } from 'react-testing-lib-wrapper';
+import ContentViewsTable from '../Table/ContentViewsTable.js';
+
+const contentViewIndex = require('./contentViewList.fixtures.json');
+
+let firstCV;
+let contentViews;
+beforeEach(() => {
+  const { results } = contentViewIndex;
+  contentViews = results;
+  [firstCV] = results;
+});
+
+const defaultProps = {
+  loadContentViewDetails: () => {},
+  detailsMap: {},
+};
+
+test('Can view content views on the screen', () => {
+  const { getByText, queryByTestId, queryByText } =
+    render(<ContentViewsTable results={contentViews} loading={false} {...defaultProps} />);
+
+  // query* functions will return the element or null if it cannot be found
+  // get* functions will return the element or throw an error if it cannot be found
+  // Loop through the CVs in the fixture and ensure they are present
+  contentViewIndex.results.map(({ name }) => {
+    expect(getByText(name)).toBeTruthy();
```

Comment:
If get* functions throw an error when the element is not found, it seems like we should always use `.toThrow()` with get*, and only use `.toBeTruthy` with query*.

Reply:
query* with `.toBeTruthy()` is in [the library's example](https://testing-library.com/docs/dom-testing-library/example-intro) so that does seem like a good change :+1: I'll update this.

I do see query* with `.toBeNull()` in the [example library](https://github.com/kentcdodds/react-testing-library-examples/blob/5b514ceacf5363e5d414dd70be2cce27f21cfc34/src/__local_tests__/mock.react-transition-group.js#L46) so I would prefer that. I don't think `.toThrow()` would be the best check as there is an off chance it could throw an error for a different reason other than the element being absent. Do you see any examples of this in the documentation?

Reply:
Right, thinking about it more it doesn't make sense to expect throws, we should just use query* if we expect truthy or null values, and get* if we expect certain contents, or need to do something else with the element.  I just wanted to ensure that we don't get errors thrown when we actually just mean for the test to fail.

Reply:
I do wish the library gave more guidance on when to use which!

Reply:
for instance, if you have 
```js
expect(getByText(name)).toBeTruthy();
```
and it's falsey, that would throw an error.  But actually you just want the test to fail.

Reply:
I just defaulted to using `queryBy*`, I think the difference is slightly different error messages, but I don't see clear guidelines on when to use which and I can't think of a good reason to error the query itself instead of checking the expected outcome with an assertion.